go.mod

```diff
@@ -2,8 +2,7 @@ module github.com/rancher/rancher
 
 go 1.21
 
-// on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+replace github.com/rancher/channelserver v0.5.1-0.20230719220800-0a37b73c7df8 => github.com/KevinJoiner/channelserver v0.0.0-20240124134223-b6d3819b0a98
```

Comment:
Remove and update commit after https://github.com/rancher/channelserver/pull/25 is merged